src/oop_ext/interface/_interface.py

```diff
@@ -287,14 +303,18 @@ def ForgetResult(self, args):
 __ImplementedInterfacesCache = __ResultsCache()
 
 
-def _CheckIfClassImplements(class_, interface):
+def _CheckIfClassImplements(class_, interface, *, requires_declaration=False):
     """
     :type class_: type or classobj
     :param class_:
         A class type (NOT an instance of the class).
 
     :type interface: Interface
 
+    :type requires_declaration: bool
+        If `True`, the Interface must have been explicitly declared through :py:func:`ImplementsInterface`
```

Comment:
For internal functions it is enough to point to the description given by one of the public APIs, like:

```rst
:type requires_declaration: bool
   See `AssertImplements`
```

Reply:
Done